qdl.c

```diff
@@ -111,6 +111,7 @@ int main(int argc, char **argv)
 	int opt;
 	bool qdl_finalize_provisioning = false;
 
+	printf("QDL version %s %s\n", VERSION, BUILD_TIME);
```

Comment:
I'd like to make the tool less noisy... How about instead adding a --version argument which prints the version and exits?